index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
+    <title>My Curriculum Vitae</title>
+  </head>
+  <body>
+    <div id="content">
+      <div class="photo">
+        <img src="photo.jpg" alt="Maria Rachkovskaya" width="200" height="200">
+      </div>
+      <div class="name">
+        <h1>Maria Rachkovskaya</h1>
+      </div>
+      <div class="contacts">
+        <h2>Contact me</h2>
+        <p>Phone: +375 (29) 114-55-86</p>
+        <p>Email: maria.rachkovskaja@gmail.com</p>
+        </div>
+      <div class="summary">
+        <h2>Summary</h2>
+        <p>Seeking a career opportunity in Frontend Development.
+            Always ready to learn something new, develop skills and make progress in work.</p>
+      </div>
+      <div class="skills">
+        <h2>Skills</h2>
+        <p>I have basic knowledge in the following areas:</p>
+        <ul type="circle">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>Java</li>
+          <li>C++</li>
+          <li>Git</li>
+          <li>Creational patterns</li>
+        </ul>
+      </div>
+      <div class="english">
+        <h2>English level</h2>
+        <p>My English level is B2.</p>
+      </div>
+      <div class="education">
+        <h2>Education</h2>
+        <p id="university">Belarusian State University of Informatics and Radioelectronics</p>
+        <p id="date">07-2016/07-2020</p>
+        <p>Bachelor in economics of e-business</p>
+        <hr>
+        <p>I completed HTML and CSS courses in Codecademy and SoloLearn.</p>
+      </div>
+      <div class="code-examples">
+        <h2>Code examples</h2>
+        <p>Here's some <a href="https://github.com/warryr">examples</a> of my code.</p>
```

Comment:
It is better that the user does not leave your site when clicking on the link. Use for this attribute target="_blank"